cmd/pr-reminder/main_test.go

```diff
@@ -849,3 +850,108 @@ func Test_hasUnactionableLabels(t *testing.T) {
 		})
 	}
 }
+
+func Test_sendMessage(t *testing.T) {
+	type args struct {
+		logger      *logrus.Entry
+		channel     string
+		prs         []prRequest
+		slackClient slackClient
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "send message to valid channel",
+			args: args{
+				logger:  logrus.NewEntry(logrus.New()),
+				channel: "valid-channel",
+				prs: []prRequest{
+					{
+						Repo:        "org/repo",
+						Number:      1,
+						Url:         "github.com/org/repo/1",
+						Title:       "Test PR",
+						Author:      "user",
+						Created:     time.Now(),
+						LastUpdated: time.Now(),
+					},
+				},
+				slackClient: fakeSlackClient{userIdsByEmail: map[string]string{}},
+			},
+			wantErr: false,
+		},
+		{
+			name: "send message with empty PR list",
+			args: args{
+				logger:      logrus.NewEntry(logrus.New()),
+				channel:     "valid-channel",
+				prs:         []prRequest{},
+				slackClient: fakeSlackClient{userIdsByEmail: map[string]string{}},
+			},
+			wantErr: false,
+		},
+		{
+			name: "send message with few PRs",
+			args: args{
+				logger:  logrus.NewEntry(logrus.New()),
+				channel: "valid-channel",
+				prs: []prRequest{
+					{
+						Repo:        "org/repo",
+						Number:      1,
+						Url:         "github.com/org/repo/1",
+						Title:       "Test PR",
+						Author:      "user",
+						Created:     time.Now(),
+						LastUpdated: time.Now(),
+					},
+					{
+						Repo:        "org/repo",
+						Number:      2,
+						Url:         "github.com/org/repo/2",
+						Title:       "Another Test PR",
+						Author:      "user2",
+						Created:     time.Now(),
+						LastUpdated: time.Now(),
+					},
+				},
+				slackClient: fakeSlackClient{userIdsByEmail: map[string]string{}},
+			},
+			wantErr: false,
+		},
+		{
+			name: "send message with more than 100 PRs",
```

Comment:
Technically you are not testing anything here. In all test cases, you are expecting an error that will never happen. The fakeslackclient will always return with no error.